### DISCUSSION TITLE
Imgui

### DISCUSSION THREAD

**Discussion by erikpa1** [bevyengine/bevy#4401] [Q&A]
Hi guys, 

for long time I'm searching for right technology for my product and I found this treasure (I mean rust and bevy). I have three js web application, but I need 3D editor, where I can create and edit my structures for three js (scene editor, edit photos, cut videos ,....). 

And my question is, did somebody tried integration bevy with **imgui** (https://github.com/imgui-rs/imgui-rs) ? Because if this is somehow possible, this is win for me.

**Comment by alice-i-cecile** [comment]
[egui](https://github.com/emilk/egui) is very similar and directly inspired, nad there's a very popular [bevy_egui](https://github.com/mvlabat/bevy_egui) integration for it :)

[bevy_editor_pls](https://github.com/jakobhellermann/bevy_editor_pls) (and the underlying [bevy_inspector_egui](https://github.com/jakobhellermann/bevy-inspector-egui)) may also help you with your more advanced editing needs.

**Reply by erikpa1** [reply]
Hi, thanks for answer, this library looks pretty young comparing to imgui, I hope it will fulfill all my requirements, thanks :)
